docs/modules/ROOT/pages/ai-services.adoc

```diff
@@ -275,6 +275,15 @@ A document retriever fetches data from an external source and provides it to the
 
 This guidance aims to cover all crucial aspects of designing AI services with Quarkus, ensuring robust and efficient interactions with LLMs.
 
+== Moderation
+
+By default, @RegisterAiService annotated interfaces don't moderate content. However, users can opt-in to having the LLM moderate
+content by annotating the method with `@Moderate`.
+
+For this to work, a `dev.langchain4j.model.moderation.ModerationModel` must be configured,
+which the extension does automatically if `@RegisterAiService(moderationModelSupplier = RegisterAiService.BeanModerationModelSupplier.class)`
```

Comment:
I am not sure what required from the user?

- A bean exposing `ModerationModel`?
- A supplier class?

Reply:
It does not

Reply:
Docs fixed